tests/Serializer/AbstractItemNormalizerTest.php

```diff
@@ -231,7 +231,12 @@ public function testNormalizeReadableLinks()
 
         $serializerProphecy = $this->prophesize(SerializerInterface::class);
         $serializerProphecy->willImplement(NormalizerInterface::class);
-        $serializerProphecy->normalize($relatedDummy, null, Argument::type('array'))->willReturn(['foo' => 'hello']);
+        $contextWithoutIri = Argument::allOf(
+            Argument::type('array'),
+            Argument::withEntry('resource_class', 'ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\RelatedDummy'),
+            Argument::not(Argument::withEntry('iri', '/dummies/1')) // child relation should not have parent iri
+        );
+        $serializerProphecy->normalize($relatedDummy, null, $contextWithoutIri)->willReturn(['foo' => 'hello']);
```

Comment:
I wanted to add a `   $iriConverterProphecy->getIriFromItem($relatedDummy)->willReturn('/dummies/2');`  but this method will never be called as it's mocked. So I just check that a wrong iri is not given.